### PR TITLE
CLOUDP-246401: Add dbusers e2e test as we have IDPs now

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

_Jira ticket:_ [CLOUDP-246401](https://jira.mongodb.org/browse/CLOUDP-246401)

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/dbusers/create.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func (opts *CreateOpts) isOIDCSet() bool {
 }
 
 func (opts *CreateOpts) isExternal() bool {
-	return opts.isX509Set() || opts.isAWSIAMSet() || opts.isLDAPSet() || opts.isOIDCSet()
```
Comment (blva) [inline internal/cli/dbusers/create.go:86]: some latest change was done to the dbusers, blocking $external value on OIDC creation, so this is fixing a bug on the OIDC DBUsers creation. cc @tibulca

File: test/e2e/atlas/dbusers_test.go [hunk 2]
```diff
@@ -133,6 +133,9 @@ func TestDBUsersWithStdin(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 
+	idpID, _ := os.LookupEnv("IDENTITY_PROVIDER_ID")
```
Comment (gssbzn) [inline test/e2e/atlas/dbusers_test.go:136]: can  you add a `require.NotEmpty` for this value?

File: test/e2e/atlas/dbusers_test.go [hunk 3]
```diff
@@ -156,8 +159,25 @@ func TestDBUsersWithStdin(t *testing.T) {
 		testCreateUserCmd(t, cmd, username)
 	})
 
+	t.Run("Create OIDC user", func(t *testing.T) {
+		require.NoError(t, err)
```
Comment (gssbzn) [inline test/e2e/atlas/dbusers_test.go:163]: seems like the wrong place for this line, as where is this error coming from?

Comment (blva) [reply test/e2e/atlas/dbusers_test.go:163]: yeah i. don't know how this got there, I'll fix!

**Review by gssbzn — DISMISSED** [review]
LGTM

**Review by tibulca — DISMISSED** [review]
LGTM

**Review by gssbzn — APPROVED** [review]

**Review by tibulca — APPROVED** [review]

**Pull request merged**